projects/swimlane/ngx-ui/src/lib/components/drawer/drawer.component.ts

```diff
@@ -7,19 +7,25 @@ import {
   HostListener,
   ViewEncapsulation,
   OnDestroy,
-  ChangeDetectionStrategy
+  ChangeDetectionStrategy,
+  TemplateRef
 } from '@angular/core';
 import { trigger } from '@angular/animations';
 
 import { drawerTransition } from '../../animations/animations';
+import { coerceNumberProperty, coerceBooleanProperty } from '@angular/cdk/coercion';
```

Comment:
these imports probably belong up with the other angular imports

Reply:
moved it up